utils/src/parallel_restore.h

```diff
@@ -46,6 +46,10 @@ struct scoutfs_parallel_restore_inode {
 	u32 gid;
 	u32 mode;
 	u32 rdev;
+#ifdef SCOUTFS_FORMAT_VERSION_FEAT_RETENTION
```

Comment:
I wouldn't do these #ifdefs here. These flags are universal for format V2 and they are going to be needed for all sorts of other flags in the future - they aren't tied to retention alone. Just like with the project ID we should always generate v2 content on disk when restoring.

So please just remove the ifdefs from this and the project ID patch, and if someone makes a wild version where it doesn't compile, they're on their own.

Reply:
The original thought for this was that these constants were guaranteed to exist on a format v2 fs where they wouldn't on fmt < v2. However, I agree they should be removed now that we only allow version v2.